### PR TITLE
Further cleanup of Gradle build files

### PR DESCRIPTION
**Pull request by msridhar**
Consolidate some version numbers in the top-level build file to make upgrading easier

### CODE REVIEW

File: returnsrcvr-qual/build.gradle [hunk 1]
```diff
@@ -14,7 +14,7 @@ sourceCompatibility = 1.8
 
 dependencies {
     // This dependency is found on compile classpath of this component and consumers.
-    implementation 'org.checkerframework:checker-qual:2.10.0'
```
Comment (mernst) [inline returnsrcvr-qual/build.gradle]: The change implementation=>api doesn't seem related to the main point of this pull request, but OK.

Comment (msridhar) [reply returnsrcvr-qual/build.gradle]: Yes true.  That was for consistency with `object-construction-qual` which makes this an api dependence.

**Comment by msridhar** [comment]
I think for this kind of cleanup we can land without two reviews, so I'll go ahead
